<launch>
    <node name="feature_tracker" pkg="feature_tracker" type="feature_tracker" output="log">
        <param name="calib_dir" type="string" value="$(find feature_tracker)/config/" />
        <param name="cam_name0" type="string" value="vi_l" />
        <param name="max_cnt" type="int" value="200" />
        <param name="min_dist" type="int" value="30" />
        <param name="window_size" type="int" value="20" />
        <param name="freq" type="int" value="2" />
        <param name="F_threshold" type="double" value="3.0" />
        <param name="T_threshold" type="double" value="0.5" />
        <param name="show_track" type="bool" value="true" />
        <param name="show_stereo" type="bool" value="false" />
        <param name="stereo_track" type="bool" value="false" />
        <param name="use_T" type="bool" value="false" />
        <param name="use_F" type="bool" value="true" />

        <remap from="~imu" to="/imu0" />
        <remap from="~raw_image" to="/cam0/image_raw" />
    </node>
</launch>
